### ISSUE TITLE
RCS1097 False Negative 4

### ISSUE THREAD

**Issue by nnpcYvIVl** [JosefPihrt/Roslynator#519]
RCS1097 isn't flagged on ```address.ToString()```. Is this a false negative?

```csharp
using System.Net;
using System.Text;

namespace RCS1097_False_Negative
{
    static class Class
    {
        static void M(IPAddress[] ipaddressarray)
        {
            StringBuilder addressList = new StringBuilder();
            foreach (IPAddress address in ipaddressarray)
            {
                addressList.Append(address.ToString());
            }
        }
    }
}
```
